File: RandomTeamChooserCs/TeamSideTool.cs
using System;
using System.Collections.Generic;

namespace RandomTeamChooser
{
    public static class TeamSideTool
    {
        private static List<string> teamSides = new List<string> { "T", "CT" };

        private static Random rnd = new Random();

        public static void ChooseTeam()
        {
            Console.Write("==== TEAM SIDE CHOOSER TOOL ====\n\n");

            int index = rnd.Next(teamSides.Count);
            string result = teamSides[index];

            Console.WriteLine(string.Format("Your Team: {0}", result));
            Console.WriteLine("\nPress enter to exit...");
            Console.ReadLine();
        }
    }
}

File: RandomTeamChooserCpp/TeamSideTool.cpp
#include "TeamSideTool.h"

void TeamSideTool::ChooseMap()
{
    std::cout << "==== TEAM SIDE CHOOSER TOOL ====\n\n";

    srand((unsigned int)time(NULL));

    int index = rand() % teamSides.size();
    std::string result = teamSides[index];

    std::cout << "Your Team: " << result << "\n\n";
    std::cout << "Press enter to exit...";
    fgetc(stdin);
    std::cin.ignore(std::numeric_limits<std::streamsize>::max(), '\n');
}